src/java.desktop/share/classes/sun/java2d/cmm/lcms/LCMSImageLayout.java

```diff
@@ -222,81 +223,67 @@ public static LCMSImageLayout createImageLayout(BufferedImage image) throws Imag
         l.height = image.getHeight();
 
         switch (image.getType()) {
-            case BufferedImage.TYPE_INT_RGB:
-            case BufferedImage.TYPE_INT_ARGB:
-            case BufferedImage.TYPE_INT_BGR:
-                do {
-                    IntegerComponentRaster intRaster = (IntegerComponentRaster)
-                            image.getRaster();
-                    l.nextRowOffset = safeMult(4, intRaster.getScanlineStride());
-                    l.nextPixelOffset = safeMult(4, intRaster.getPixelStride());
-                    l.offset = safeMult(4, intRaster.getDataOffset(0));
-                    l.dataArray = intRaster.getDataStorage();
-                    l.dataArrayLength = 4 * intRaster.getDataStorage().length;
-                    l.dataType = DT_INT;
-
-                    if (l.nextRowOffset == l.width * 4 * intRaster.getPixelStride()) {
-                        l.imageAtOnce = true;
-                    }
-                } while (false);
```

Comment:
Any idea why the "do/while(false)" was used here?

Reply:
Probably it is just a way to highlight and improve readability of a block of code, but simply `{}` would be enough in this case.